packages/plugin-playground/package.json

```diff
@@ -42,9 +42,10 @@
     "rspack-plugin-virtual-module": "0.1.13"
   },
   "devDependencies": {
-    "@babel/types": "^7.26.7",
-    "@rsbuild/plugin-react": "~1.1.0",
-    "@rslib/core": "0.5.2",
+    "@babel/types": "^7.27.1",
+    "@rsbuild/plugin-react": "~1.3.1",
+    "@rslib/core": "~0.6.9",
+    "@rspress/plugin-playground": "workspace:*",
```

Comment:
```suggestion
```

It seems comes from https://github.com/web-infra-dev/rspress/blob/9a842585e11a12adbcca93154a023575b4202635/packages/plugin-playground/package.json#L47, I don't know whether this is intended.

cc @SoonIter